main/guides/coreeval/local-testnet.md

```diff
@@ -0,0 +1,33 @@
+# Create a Local Testnet
+
+To create, start, and deploy an Agoric contract to a local Agoric Testnet, do the following:
+
+
+1. Install *Go* if needed.
```

Comment:
```suggestion
1. Install *Go* if needed. For example:
```

Installing `go` with `brew` assumes a mac with `brew` installed. Our [platform section](https://docs.agoric.com/guides/getting-started/#platform-linux-shell-or-equivalent) says we support linux, MacOS, and WSL.

Building the `go` stuff is something we left out of the SDK 'till now. It merits its own section. In [Getting Started](https://docs.agoric.com/guides/getting-started/). The idea was that it wasn't needed for smart contract development. With the smart wallet architecture, this is no longer really viable.

Reply:
p.s. the https://docs.agoric.com/guides/dapps/starting-multiuser-dapps.html looks like it discusses building the `go` stuff.

Reply:
I've moved the installation of Go to the Getting Started section in another PR.

Also, I changed the installation instructions so they're platform agnostic. (I hadn't realized that "brew" was Mac-specific.)